app/controllers/ExclusionListController.scala

```diff
@@ -264,7 +264,7 @@ class ExclusionListController @Inject()(
     individualSelectionOption: Option[String])(implicit request: AuthenticatedRequest[_]): Result =
     listOfMatches.size match {
       case 0 =>
-        Logger.error("Matches are zero size")
+        Logger.error("[ExclusionListController][searchResultsHandleValidResult] List of mathces is empty")
```

Comment:
We see this error a bit in prod, is this something you'd view as an error level log or is it expected behaviour? Also, "matches" > "matches"

Reply:
I couldn’t honestly say if this is expected behaviour or not